uber/models.py

```diff
@@ -367,13 +376,14 @@ def __getattr__(self, name):
             return suffixed
 
         try:
-            [multi] = [col for col in self.__table__.columns if isinstance(col.type, MultiChoice)]
             choice = getattr(c, name)
-            assert choice in [val for val, desc in multi.type.choices]
+            if len(self.multichoice_columns) == 1:
+                multi = self.multichoice_columns[0]
+                if choice in multi.type.choices_dict:
+                    return choice in getattr(self, multi.name + '_ints')
         except:
```

Comment:
Hmm, this is a lot of stuff inside the `try` statement. Since you're only expecting the exception in one case (as indicated by the comment), would it be better to expressly test for that? Like:

```
choice = getattr(c, name, None)
if choice is None:
   [the rest of the block]
```

Or just try/except the `choice = getattr(c, name)` explicitly and test for the exception that getattr() raises (which I don't know off the top of my head), then put the rest under `else`.